doc/classes/AnimationNodeTransition.xml

```diff
@@ -16,43 +16,49 @@
 			<return type="int" />
 			<param index="0" name="caption" type="String" />
 			<description>
+				Returns input index which corresponds to [param caption]. If not found, returns [code]-1[/code].
 			</description>
 		</method>
 		<method name="get_input_caption" qualifiers="const">
 			<return type="String" />
 			<param index="0" name="input" type="int" />
 			<description>
+				Returns name of input at given index. This name shows in editor next to node input.
 			</description>
 		</method>
 		<method name="is_input_set_as_auto_advance" qualifiers="const">
 			<return type="bool" />
 			<param index="0" name="input" type="int" />
 			<description>
+				Returns [code]true[/code] if auto-advance is enabled on given input.
 			</description>
 		</method>
 		<method name="set_input_as_auto_advance">
 			<return type="void" />
 			<param index="0" name="input" type="int" />
 			<param index="1" name="enable" type="bool" />
 			<description>
+				Enables or disables auto-advance for given index. If enabled, state changes to the next input after playing animation once. If enabled for the last input state loops to the first.
 			</description>
 		</method>
 		<method name="set_input_caption">
 			<return type="void" />
 			<param index="0" name="input" type="int" />
 			<param index="1" name="caption" type="String" />
 			<description>
+				Sets name of input at given index. This name shows in editor next to node input.
 			</description>
 		</method>
 	</methods>
 	<members>
 		<member name="enabled_inputs" type="int" setter="set_enabled_inputs" getter="get_enabled_inputs" default="0">
-			The number of enabled input ports for this node.
+			The number of enabled input ports for this node. Maximum is [code]31[/code].
```

Comment:
```suggestion
			The number of enabled input ports for this node. The maximum index is [code]31[/code], i.e. all 32 enabled input ports are enabled.
```

Reply:
This doesn't refer to an index, it refers to the number of inputs, which actually is 31 (due to a bug, refer discussion above).

Reply:
Then it should be changed to start at one and not be limited to 31, but 32. This should be fixed instead of documenting the bug.

Reply:
Though I see it's tracked in #60307 so it's probably fine for now, we should just remember to update the docs when #60307 is solved.

Reply:
That is true, however it's out of scope for this pr. I thought it's better to document the current behaviour, and update documentation when the bug gets fixed. Or is it better to leave this fact out altogether for now?

Reply:
In the end I think it's fine to document it for now, yes.